iam/snippets/snippets_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package snippets
+
+import (
+	"math/rand"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func TestViewGrantableRoles(t *testing.T) {
+	projectID := os.Getenv("GOLANG_SAMPLES_PROJECT_ID")
```

Comment:
Please use `testutil` to get the project ID. Example: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/monitoring/snippets/channel_delete_test.go#L22

Reply:
This gives an error because I'm working in a fork and can't reference the internal main package in the main repo. I don't want to reference my fork because that'll be wrong once the PR is merged. What's a good workaround?

Reply:
You should have a single repo locally and a different remote for your fork/the origin. See https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#contributing. Then, you'll be able to import the `testutil` package.